File: benchmark/2026-09-17-7677581(latest)/semantic-compression/_superseded_run1/probes/python/F05.P1.py
def f(items: list[int]) -> None:
    items[0] = 99


def call_f() -> int:
    # BEGIN PROBE F05.P1
    x = [1, 2, 3]
    f(x)
    return x[0]
    # END PROBE F05.P1


print(call_f())

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/typescript/F05.P1.ts
function f(v: number[]): void {
  v[0] = 99
}

function probe(): number {
// BEGIN PROBE F05.P1
const x = [1, 2, 3]
f(x)
return x[0]
// END PROBE F05.P1
}

console.log(probe())
